tensornetwork/linalg/tests/test_krylov.py

```diff
@@ -0,0 +1,298 @@
+import numpy as np
+from jax import config
+import pytest
+import tensornetwork
+import tensornetwork.linalg.krylov
+import tensornetwork.linalg.initialization
+from tensornetwork.tests import testing_utils
+
+# pylint: disable=no-member
+config.update("jax_enable_x64", True)
+sparse_backends = ["numpy", "jax"]
```

Comment:
symmetric backend also supports certain sparse operations.

Reply:
please add tests for those as well

Reply:
I think it's going to take a little bit of work before tn.Tensor more generally functions properly with Symmetric; for example, it looks like tn.ones fails. I don't know how to fix any of this stuff because the error messages ("int" has no attribute "flat flows") are pretty inscrutable to me. I would recommend we address this issue separately.

Reply:
The first few things I can think of where BlockSparseTensors are different are
*  initialization
* elementwise access
* slicing
Other than that, we should make sure that tf.Tensor should works with `BlockSparseBackend`.

Reply:
I think it's going to be difficult to ensure this until we sort out initialization, is the thing.

I would also recommend we subclass tf.Tensor for the sparse case, so that stuff about the sparse data structure be accessible directly

Reply:
As long as everything operates on the backend level, it should be fine.